example/app.json

```diff
@@ -4,8 +4,8 @@
     "description": "A cross-platform ActionSheet component. https://github.com/expo/react-native-action-sheet",
     "slug": "react-native-action-sheet-example",
     "privacy": "public",
-    "sdkVersion": "25.0.0",
-    "version": "1.0.0",
+    "sdkVersion": "32.0.0",
+    "version": "0.0.1",
```

Comment:
I think I would keep this at version 1 or bump this to 1.1.0 or even 2.0.0 my mind escapes me 😅

Reply:
I think the next npm version is more appropriate!
So, I will try to set `3.0.0` (This PR brings the breaking change in this package,  so maybe jump up major version), is it no problem?